plugins/CMakeLists.txt

```diff
@@ -1,76 +1,81 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
-if(WASMEDGE_PLUGIN_WASI_NN_BACKEND)
+if (WASMEDGE_PLUGIN_WASI_NN_BACKEND)
```

Comment:
You should never change the coding style here. There is no space between `if` and `(`.